__tests__/app.test.js

```diff
@@ -331,3 +331,24 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+
+
+
+describe.only("GET /api/users", () => {
+  test("GET:200 responds with an array of user objects", () => {
+       return request(app)
+         .get("/api/users")
+         .expect(200)
+         .then((response) => {
+           const users = response.body.users;
+           expect(users.length).toBe(4);
+           users.forEach((user) => {
+             expect(typeof user).toBe("object");
```

Comment:
.objectContaining or .toMatchObject would be benficial to use here to check both the keys and values